DataStructure2/assignment_2/CollegeCounselling.java

```diff
@@ -0,0 +1,251 @@
+
+					// assign college to candidate
+					assignedColleges.put(candidateName, collegesList.getElement(userInput - 1).getCollegeName());
+
+					// reduce the number of seats in that college by 1
+					--numberOfSeats;
+
+					// create a new object with the updated seats
+					College college = collegesList.getElement(userInput -1 );
+
+					college.setCollegeSeats(numberOfSeats);
+
+					// overwrite the list with new object
+					collegesList.overwriteElement(userInput - 1, college);
+
+				}
+
+				// if number of seats is less than or equal to zero then display message
+				else
+					System.out.println("The college you entered is already full. Please select another college");
+
+			}
+
+		}
+		catch(Exception ex)
+		{
+
+			System.out.println("Something went wrong: "+ex.getMessage());
+		}
+		return assignedColleges;
+	}
+
+
+	public ArrayList<College> createListOfColleges()
+	{
+		ArrayList<College> collegeLists = new ArrayList<College>();
+		try
+		{
+
+			College college1 = new College("LNMIIT", 1,1);
+			College college2 = new College("Amity University", 2,1);
+			College college3 = new College("JECRC",3,2);
+			College college4 = new College("SKIT",4,3);
+			College college5 = new College("Poornima University", 5,3);
+			College college6 = new College("Jaipur University", 6,2);
+
+			collegeLists.add(college1);
+			collegeLists.add(college2);
+			collegeLists.add(college3);
+			collegeLists.add(college4);
+			collegeLists.add(college5);
+			collegeLists.add(college6);
+			
+		}
+		catch(Exception ex)
+		{
+			System.out.println("Something went wrong: "+ex.getMessage());
+			createListOfColleges();
+		}
+
+		return collegeLists;
+	}
+
+	public Queue<Candidate> createQueueOfCandidates()
+	{
+		Queue<Candidate> candidates = new Queue<Candidate>();
+		try
+		{
+
+			Candidate candidate1 = new Candidate("test1", 1);
+			Candidate candidate2 = new Candidate("test2", 2);
+			Candidate candidate3 = new Candidate("test3", 3);
+			Candidate candidate4 = new Candidate("test4", 4);
+			Candidate candidate5 = new Candidate("test5", 5);
+			Candidate candidate6 = new Candidate("test6", 6);
+			Candidate candidate7 = new Candidate("test7", 7);
+			Candidate candidate8 = new Candidate("test8", 8);
+			Candidate candidate9 = new Candidate("test9", 9);
+			Candidate candidate10 = new Candidate("test10", 10);
+			Candidate candidate11 = new Candidate("test11", 11);
+			Candidate candidate12 = new Candidate("test12", 12);
+			Candidate candidate13 = new Candidate("test13", 13);
+
+			candidates.enqueue(candidate1);
+			candidates.enqueue(candidate2);
+			candidates.enqueue(candidate3);
+			candidates.enqueue(candidate4);
+			candidates.enqueue(candidate5);
+			candidates.enqueue(candidate6);
+			candidates.enqueue(candidate7);
+			candidates.enqueue(candidate8);
+			candidates.enqueue(candidate9);
+			candidates.enqueue(candidate10);
+			candidates.enqueue(candidate11);
+			candidates.enqueue(candidate12);
+			candidates.enqueue(candidate13);
```

Comment:
for(int i = 1; i< 13; i++)
{
      candidates.enqueue(new Candidate("test"+i, i));
}